src/views/components/trace/trace-chart-table/trace-item.vue

```diff
@@ -0,0 +1,210 @@
+<template>
```

Comment:
![image](https://user-images.githubusercontent.com/19775780/62281612-a810d600-b480-11e9-9c80-8562f4b2a9df.png)

warning need fix

Reply:
use <script lang='js'>  not<script type="text/javascript"> to fix